addons/colorblind/addon.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "Color blindness accommodations",
+  "description": "A few features to help make Scratch more accessible to colorblind users.",
```

Comment:
We can keep the words "color blindness" in the description so that the old name of the addon is still searchable.

Reply:
Is that really necessary? The addon was never released with the previous name.

Reply:
It is not necessary